### ISSUE TITLE
Gameplay cursor should disappear on idle

### ISSUE THREAD

**Issue by nl-tatatat** [ppy/osu#2901]
The cursor should be hidden during osu!taiko gameplay (and osu!catch and osu!mania in the future). It only  serves as a distraction outside of clicking the skip button, and clicking the button at the bottom right of the screen. Unless the cursor is near that button, it should be hidden.

**Comment by Hickacou** [comment]
I think it could be a good idea to hide it when it's inactive and make it visible when we move it and during the breaks.

**Comment by nl-tatatat** [comment]
that seems rather reasonable.

**Comment by peppy** [comment]
Yep, similar behaviour to stable sounds great. The cursor should hide after an inactivity period when in gameplay and the ruleset is not providing a cursor.

**Comment by Joehuu** [comment]
Resolved in https://github.com/ppy/osu/pull/20713.

**Comment by Joehuu** [comment]
Actually, this also tracks the cursor hiding when inactive on breaks on the osu! ruleset, so reopening (was just looking at the OP description / the other rulesets).
